src/Session.php

```diff
@@ -1004,13 +1004,15 @@ public static function checkLoginUser()
 
     /**
      * Get the name of the right.
+     * This should only be used when it is expected that the request going to be terminated.
+     * The session will be closed by this method.
      *
-     * This only works for well-known rights.
+     * @param string $module The module
      * @param int $right The right
      * @return string The right name
      * @internal No backwards compatibility promise. Use in core only.
      */
-    public static function getRightNameForError(int $right): string
+    public static function getRightNameForError(string $module, int $right): string
```

Comment:
Changing signature in a bugfixes version may break some plugins. Either we have to create a new method (e.g. `public static function getRightLabel(string $module, int $right): string`, either we have to make the new param optional:
```suggestion
    public static function getRightNameForError(int $right, ?string $module = null): string
```

Reply:
Even when the PHPDoc has always said there was no BC promise and it was only meant to be used in the core?

Reply:
PHPDoc was pretty clear about this, so OK to change the signature.